docs/Users_Guide/overview.rst

```diff
@@ -67,7 +69,8 @@ their verification environments. Wrappers can be run individually, or as
 a group of wrappers that represent a sequence of MET processes. New
 wrappers can readily be added to the METplus Wrappers package due to
 this modular design. Currently, METplus Wrappers can easily be applied
-by any user on their own computer platform that supports Python 3.8.  We have deprecated support to Python 2.7.
+by any user on their own computer platform that supports Python 3.10.
```

Comment:
Should `|python_version|` be used here instead of hard-coding 3.10? Also, line 73 noting deprecated support for 2.7 could be removed.

Reply:
I think using |python_version| here would be a great idea.  (my 2 cents)

Reply:
Sounds good, @lisagoodrich. Can you make that change?

Reply:
Thank you for your patience @georgemccabe. I have added "|python_version|" to line 72 and 
removed this: "We have deprecated support to Python 2.7."

Should updating  "|python_version|"  and  "|met_version|" be added to the Release Notes?  (If you'd like me to do this, please point me to the correct section.)

Are there any other changes you would like me to make?

Reply:
The Python version doesn't need to be updated for every release, so that isn't necessary to add to the release guide. It would be nice to automate the update of the MET version, but until that is done it should be updated as part of the release guide instructions. There isn't a clear place to put this in the release guide that doesn't involve moving things around, so it may be best to just create a GitHub issue to automate the update of the MET version in the documentation and assign it to me.